external/model-preparation-algorithm/tests/ote_cli/test_segmentation.py

```diff
@@ -50,18 +50,18 @@
         '--learning_parameters.num_iters',
         '2',
         '--learning_parameters.batch_size',
-        '2'
+        '4'
     ]
 }
 
-root = '/tmp/ote_cli_seg/'
+root = '/tmp/ote_cli/'
 ote_dir = os.getcwd()
 
-templates = Registry('external/model-preparation-algorithm', experimental=True).filter(task_type='SEGMENTATION').templates
+templates = Registry('external/model-preparation-algorithm').filter(task_type='SEGMENTATION').templates
```

Comment:
Hmm, lite-hrnet-18 will be collected. (not mod-2). How do you think about this?

Reply:
I'm sorry I didn't understand your comment.

Reply:
Please refer here : https://github.com/openvinotoolkit/training_extensions/tree/develop/external/model-preparation-algorithm/configs/segmentation/ocr-lite-hrnet-18
Lite-hrnet-18/template.yaml is reverted. From this, lite-hrnet-18 will be collected as template even though we are keeping only for compatibility purpose.

Reply:
I think compatibility can be established by passing this test necessarily. What do you think?

Reply:
Also, I know that the template, which is experimental, is not currently in the segmentation config.